packages/db/src/schema/openbanking.ts

```diff
@@ -1,166 +1,169 @@
 import { relations, sql } from "drizzle-orm";
 import {
   bigint,
-  boolean,
+  float,
   index,
   json,
   mysqlEnum,
   timestamp,
-  unique,
   uniqueIndex,
   varchar,
 } from "drizzle-orm/mysql-core";
 
-import { ConnectorEnv, ConnectorStatus, ConnectorType } from "../enum";
+import { BalanceType } from "../enum";
 import { mySqlTable } from "./_table";
+import { currency } from "./currency";
+import { resource } from "./provider";
 
-export const countryCodes = mySqlTable(
-  "countryCodes",
+export const account = mySqlTable(
+  "account",
   {
-    id: bigint("id", { mode: "number" }).primaryKey().autoincrement(),
+    id: bigint("id", { mode: "bigint" }).primaryKey().autoincrement(),
     createdAt: timestamp("created_at")
       .default(sql`CURRENT_TIMESTAMP`)
       .notNull(),
     updatedAt: timestamp("updated_at").onUpdateNow(),
 
-    code: varchar("code", { length: 2 }).notNull(),
-    active: boolean("active").default(true),
+    resourceId: bigint("resource_id", { mode: "bigint" }).notNull(),
```

Comment:
For manual data entry to work, the account table must be able to be populated even without "resourceId" so this field must be nullable. Let me explain further: since the account is the main entrypoint, it must be able to be tied to a resource or not. We only have the resource the moment we create a connection with a third-party provider. For manual entry, I don't necessarily have to create a resource first, but I do have to be able to create an account with related balance and related transactions. Let's talk in private maybe I can explain better verbally.